rachel-cunningham/project-ai-investment-playground
# User CRUDL functions

Changes:
- /users/:username .put and .delete route methods in users.router
- update and deleteUser functions in users.controller and their counterparts in users.service
- createUser, updateUser, and deleteUser functions in api.js
- added a mess of stuff to ExampleLoginPage to test out api calls
- some other random little changes here and there that I wish I could remember

To test:
just start up the backend and frontend and make sure all of your stuff still works on there the way you want it to
